jena-core/src/main/java/org/apache/jena/mem/NodeToTriplesMapBase.java

```diff
@@ -19,6 +19,9 @@
 package org.apache.jena.mem;
 
 import java.util.*;
+import java.util.function.Consumer;
```

Comment:
Unused import.

Reply:
As far as I can see, there is no unused import here.
The `Consumer `ist used in `org.apache.jena.util.iterator.NiceIterator#forEachRemaining`.